src/graphql/template/createQuestionTemplateRelation.graphql

```diff
@@ -2,7 +2,7 @@ mutation createQuestionTemplateRelation(
   $templateId: Int!
   $questionId: String!
   $topicId: Int!
-  $sortOrder: Int!
+  $sortOrder: Float!
```

Comment:
Actually, these files are used in generating the `src/generated`, the generator can generate the TypeScript types using the GraphQL graphs on the backend, and using the files in `src/graphql` we tell the generator which queries/mutations we want to actually use, so it can generate the required types/calls and also check if our queries/mutations are actually typesafe from the Graphql point of view.
If you try to access some fields which don't exist, or you use wrong types in the parameters the generator will fail.